packages/headless/src/features/case-assist/case-assist-analytics-actions-loader.ts

```diff
@@ -33,9 +33,12 @@ export interface CaseAssistAnalyticsActionCreators {
   /**
    * Creates a Case Assist event for when the user moves to the next stage.
    *
+   * @param stageName - The current stage name.
    * @returns A dispatchable action.
    */
-  logCaseNextStage(): AsyncThunkAction<
+  logCaseNextStage(
+    stageName?: string
```

Comment:
Is it possible that more options will be added to this in the future? If yes, having an object of options instead will allow to add parameters more easily.